Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -232,6 +232,12 @@ stretched) then you will have 2 replicas per datacenter where each replica ends
     !!! note
         A value of 0 disables the quota.
 
+* `pgs`: Set pg numbers to pool. See the [ceph documentation](https://docs.ceph.com/en/latest/rados/operations/pools/#create-a-pool) for more info. If the autosacale mode is on, the pg numbers will be set to default value, the assigned value maybe invalid.
```

Comment:
Since autoscale is always on, here is a suggestion on the wording. 
```suggestion
* `pgs`: Set a custom PG count for the pool. By default, the PGs will be managed by the Ceph PG autoscaler. See the [Ceph documentation](https://docs.ceph.com/en/latest/rados/operations/pools/#create-a-pool) for more info. 
```